carladam/petrinet/petrinet.py

```diff
@@ -237,20 +237,16 @@ def _transition_is_enabled(self, marking: PMarking, transition: Transition) -> b
         place: Place
         for arc in self.node_inputs.get(transition, ()):
             place = cast(CompletedArcPT, arc).src
-            colors: ColorSet = Counter(token.color for token in marking.get(place, set()))
-            if frozenset(arc.weight) - frozenset(colors):
-                # Arc weight specifies more colors than Place has.
-                return False
-            if any(color not in arc.weight or arc.weight[color] > count for color, count in colors.items()):
-                # Arc weight specifies more of some color token than place contains.
```

Comment:
This will cause a merge conflict with #11 so let's get that one merged first, then I'll fix this one.